superset/views/core.py

```diff
@@ -518,9 +518,10 @@ class SliceAsync(SliceModelView):  # noqa
 
 class SliceAddView(SliceModelView):  # noqa
     list_columns = [
-        'id', 'slice_name', 'slice_link', 'viz_type',
-        'datasource_link', 'owners', 'modified', 'changed_on']
-    show_columns = list(set(SliceModelView.edit_columns + list_columns))
```

Comment:
@williaster do you remember why you removed `show_columns`? Not a huge deal if there's a reason to remove this. I used it for some dashboard builder stuff outside of superset codebase.

Reply:
hmm I don't remember doing this, @graceguo-supercat did you?

Reply:
looks like here https://github.com/apache/incubator-superset/pull/4849/files#diff-ac978615a4f22c4fad4d01f39e1d4595L522